GLMakie/src/screen.jl

```diff
@@ -346,7 +347,9 @@ function apply_config!(screen::Screen, config::ScreenConfig; start_renderloop::B
     ShaderAbstractions.switch_context!(glw)
     GLFW.SetWindowAttrib(glw, GLFW.FOCUS_ON_SHOW, config.focus_on_show)
     GLFW.SetWindowAttrib(glw, GLFW.DECORATED, config.decorated)
-    GLFW.SetWindowAttrib(glw, GLFW.FLOATING, config.float)
+    if GLFW.GetPlatform() != GLFW.PLATFORM_WAYLAND
+        GLFW.SetWindowAttrib(glw, GLFW.FLOATING, config.float)
```

Comment:
Is floating not supported on Wayland?

Reply:
Apparently not. If I leave that line in place, I get the following error.
```
julia> Figure()
Error showing value of type Figure:
ERROR: GLFWError (65548): Wayland: Platform does not support making a window floating
Stacktrace:
...
```
I haven't found any mention in GLFW's docs saying this isn't supported on Wayland, but given what I know of Wayland I'm also not surprised it's not a thing applications can do — the window manager instead has control of something like always-on-top behavior.